src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/CaptureFarmingGear.kt

```diff
@@ -49,6 +49,7 @@ class CaptureFarmingGear {
         "uniquevisitors.tierprogress",
         ".* §e(?<having>.*)§6/(?<total>.*)"
     )
+    private val petMenuPattern by patternGroup.pattern("garden.fortuneguide.petsmenu", "Pets(?: \\(\\d+/\\d+\\) )?")
```

Comment:
```suggestion
    private val petMenuPattern by patternGroup.pattern(
        "petsmenu", 
        "Pets(?: \\(\\d+/\\d+\\) )?"
    )
```
You still need to change the pattern key